app/lib/specialist_publisher.rb

```diff
@@ -37,6 +38,8 @@ def document_types
     "vehicle_recalls_and_faults_alert" => VehicleRecallsAndFaultsAlertObserversRegistry,
   }.freeze
 
+  private
```

Comment:
[This shouldn't be indented](https://github.com/alphagov/styleguides/blob/master/ruby.md#classes).